src/codeflare_sdk/cluster/cluster.py

```diff
@@ -217,27 +218,38 @@ def status(
 
         return status, ready
 
+    def is_dashboard_ready(self) -> bool:
+        try:
+            response = requests.get(self.cluster_dashboard_uri(), timeout=5)
+            if response.status_code == 200:
+                return True
+        except requests.RequestException:
+            pass
+        return False
```

Comment:
Does this need to be a try/except? requests.get() should always return some status_code, right? 
Can we change it to be a bit simpler:

```python
response = requests.get(self.cluster_dashboard_uri(), timeout=5)
if response.status_code == 200:
        return True
else:    
        return False
```